entrypoints/services/MerchOrderService.ts

```
@@ -0,0 +1,166 @@
+    private static readonly SCHEMA_VERSION = 1;
+
+    /**
+     * Create a new merchant order
+     */
+    static async create(
+        order: Omit<MerchOrder, "id" | "schemaVersion">
+    ): Promise<MerchOrder> {
+        try {
+            const newOrder: MerchOrder = {
+                ...order,
+                schemaVersion: this.SCHEMA_VERSION,
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)